### ISSUE TITLE
[Bug] Crash with malformed Markdown Link (and possible vulnerability) 

### ISSUE THREAD

**Issue by ArthurGamerHD** [Artemis-RGB/Artemis#852]
Artemis crashes if you create a malformed URI, like an Email for example:

https://github.com/user-attachments/assets/dab79a19-44da-46e4-9223-97a6b02159f4

and this also opens programs on the client, if the URI is pointed to a File, like "Calc.exe"

https://github.com/user-attachments/assets/80bf6e5b-6b4f-498f-99eb-1f8285df8831

is mostly harmless since doesn't parse arguments (so, no CMD or PowerShell pesky commands), but can be used to run static VBS codes on the user machine if you provide the file somewhere, like on the Plugin package itself as demo here

https://github.com/user-attachments/assets/71e8e7f6-a8ad-4ceb-bd1d-6cabb724e186

Log:

```log
2024-07-12 12:54:22.377 [FTL] [Artemis.UI.Windows.Program] Fatal exception, shutting down
System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'shorter.metrics0b@icloud.com' with working directory 'C:\Program Files\Artemis'. The system cannot find the file specified.
   at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Markdown.Avalonia.Utils.DefaultHyperlinkCommand.GoTo(String url)
   at Markdown.Avalonia.Utils.DefaultHyperlinkCommand.Execute(Object parameter)
   at Markdown.Avalonia.Markdown.<TreatsAsHref>b__81_1(String urlTxt)
```

**Comment by RobertBeekman** [comment]
Hi there,

This is quite a nasty issue with the default link handling of the Markdown library I use, it just runs whatever your put in it.

Luckily overriding this behavior is quite straightforward. I've implemented a handler that only opens web URLs (http:// or https://) and Artemis URLs (artemis://). I looked at it briefly but couldn't find a reliable way to make mailto: links work, so those are ignored currently, just like anything else.

I was working on some workshop improvements but I believe this issue warrants an immediate update. 
Thanks for the detailed report! 

Fixed in [1.20224.07712.2](https://artemis-rgb.com/releases/windows/8b8322cc-1c45-4313-84db-16b1755b50e6)
